server/src/main/java/org/elasticsearch/common/settings/IndexScopedSettings.java

```diff
@@ -148,7 +148,6 @@ public boolean isPrivateSetting(String key) {
         switch (key) {
             case IndexMetadata.SETTING_CREATION_DATE:
             case IndexMetadata.SETTING_INDEX_UUID:
-            case IndexMetadata.SETTING_VERSION_CREATED:
```

Comment:
`VERSION_CREATED` was added to this filter in https://github.com/crate/crate/commit/5a3fa0f682f799dfdf268a24ef75ef8c5d669680. But all tests added together still passed without the setting here.